go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	github.com/bufbuild/connect-go v1.5.1
+	github.com/bufbuild/connect-grpchealth-go v1.0.1-0.20230503165053-2a56ef052862
```

Comment:
Before we merge this branch, we should cut a release of connect-grpchealth-go with the most recent two PRs, so that we can have a proper released version for this dependency.

Reply:
Could we factor that test out into a separate module inside `internal`?

Reply:
I didn't think a test-only dependency would be objectionable-- the Go tool should only actually download it if its checksum can't be found in the checksum server or if one is actually running the tests (also, this repo is teeny tiny).

But sure, I can do that. I was being "lazy programmer" since setting up a second module and a workspace can be a little bit of a hassle (and since dependabot doesn't really understand workspaces in multi-module repos).

Reply:
Looks like dependabot won't have an issue with this repo. I'm not certain under what conditions workspaces requires a `go.work.sum`, but that's the file that dependabot fails to update (and the Go tool has decided that one is not needed for this repo 🤷).

This is addressed over in #49, in 0df528356d1aca3c1b57b7cb9da1b4d2f907016e